fr.handbrake.ghb.Plugin.dvdcss.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "fr.handbrake.ghb.Plugin.dvdcss",
+    "branch": "1",
+    "runtime": "fr.handbrake.ghb",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//46",
+    "build-extension": true,
+    "separate-locales": false,
+    "build-options": {
+        "prefix": "/app/extensions/dvdcss"
+    },
+    "modules": [
+        {
+            "name": "libdvdcss",
+            "config-opts": [
+                "--enable-shared",
+                "--disable-static"
+            ],
+            "post-install": [
+                "install -Dp -m 644 fr.handbrake.ghb.Plugin.dvdcss.metainfo.xml --target-directory=${FLATPAK_DEST}/share/metainfo"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "fr.handbrake.ghb.Plugin.dvdcss.metainfo.xml"
+                },
+                {
+                    "type": "git",
+                    "url": "https://code.videolan.org/videolan/libdvdcss.git",
```

Comment:
usually the actual source (this) is first.

Reply:
Thanks. Done.